### DISCUSSION TITLE
Adding gidgethub to GitHub's documentation

### DISCUSSION THREAD

**Discussion by Mariatta** [brettcannon/gidgethub#144] [Show and tell]
I created a PR to add gidgethub to the GitHub's list of third party Python libraries https://github.com/github/docs/pull/2226

**Comment by brettcannon** [comment]
😄 I wrote a note to myself this weekend to do this today. Thanks! Did their blog post on ghapi make you want to do this, too?

**Reply by Mariatta** [reply]
While working on https://github.com/brettcannon/gidgethub/pull/145 I came across the documentation, and just wondered "why isn't gidgethub on that list?" 😛

**Comment by Mariatta** [comment]
This is now live! 😎  https://docs.github.com/en/free-pro-team@latest/rest/overview/libraries#python
